pkg/validator/url_fetcher.go

```diff
@@ -0,0 +1,30 @@
+package validator
+
+import (
+	"fmt"
+	"io"
+	"net/http"
+)
+
+// URLFetcher fetches content from a URL.
+type URLFetcher struct {
+	URL string
+}
+
+func (u *URLFetcher) Fetch() ([]byte, error) {
```

Comment:
We should not write a new implementation. Everywhere else we use go-getter. At least we should be deliberate an out creating a new interface. We fetch URL's all over the place.